src/components/profile/ProfileForm.jsx

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { TextInputField, Alert } from "evergreen-ui";
+import { TextInputField, Alert, Text, Link } from "evergreen-ui";
```

Comment:
It's better to use the `Text` in the `components/ui/text/Text` so that we can avoid having direct dependency to the external lib
We perhaps need to convert all of them (UI component) in the future